src/snowflake/cli/api/config.py

```diff
@@ -340,9 +340,18 @@ def _get_envs_for_path(*path) -> dict:
     }
 
 
-def _dump_config(conf_file_cache: Dict):
+def _dump_config(whole_conf_cache: Dict):
+    config_toml_dict = whole_conf_cache.copy()
```

Comment:
Can we replace `whole` with something like `config_and_connections`?

Reply:
Done :)